chatAppTemplate_2/js/index.js

```diff
@@ -0,0 +1,49 @@
+(function() {
+  var clearResizeScroll, sendMessage, months;
+
+
+  conf = {
+    cursorcolor: "#696c75",
+    cursorwidth: "4px",
+    cursorborder: "none"
+  };
+
+  lol = {
+    cursorcolor: "#cdd2d6",
+    cursorwidth: "4px",
+    cursorborder: "none"
+  };
+
+  months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
+
+  clearResizeScroll = function() {
+    $("#text").val("");
+    $(".messages").getNiceScroll(0).resize();
+    return $(".messages").getNiceScroll(0).doScrollTop(999999, 999);
+  };
+
+  sendMessage = function() {
```

Comment:
the append method that is adding it to the screen includes the formatting stuff because we are appending it to the html. So it needs that crap to allow it to be put on the html page correctly